### DISCUSSION TITLE
AttributeError: 'Report' object has no attribute 'publish'

### DISCUSSION THREAD

**Discussion by sdhilip200** [datapane/datapane#198] [Help] [answered]
Hi

I am getting this error when I am trying to publish the report

"AttributeError: 'Report' object has no attribute 'publish'"

Can anyone advise what would be the issue?

**Comment by Buroni** [comment, marked as answer]
Hey @sdhilip200, the method has been renamed to `Report.upload` in newer versions of Datapane. Let me know if that fixes your problem
